websites/site_config_api.py

```diff
@@ -116,10 +116,30 @@ def find_item_by_name(self, name: str) -> Optional[ConfigItem]:
                 return config_item
         return None
 
-    def generate_item_metadata(self, name: str, cls: object = None, **kwargs) -> Dict:
-        """Generate a metadata dict with blank keys for the specified item"""
+    def generate_item_metadata(
+        self,
+        name: str,
+        cls: object = None,
+        use_defaults=False,
```

Comment:
The only two places where `generate_item_metadata` is used are:
- `create_gdrive_resource_content`, creating gdrive resources
- the `update_content_metadata` management command.

It seems to me that we probably always want `use_defaults=True`, but I am not entirely sure.

What do others think? If so, I could remove the kwarg and just always use the defaults.